### ISSUE TITLE
Game launched via Playnite not detected by Discord

### ISSUE THREAD

**Issue by Kh0ldstare** [JosefNemec/Playnite#1209]
From Kholdstare#0397 on Discord

Game involved in this issue: Need for Speed Underground 2

The Discord overlay works just fine when I launch the game from Windows Explorer. I get the "Press CTRL + D" message upon startup.

But when I launch the game via Playnite, Discord doesn't seem to detect the game. I don't get the overlay message when the game starts and my hotkey for it doesn't work even though my hotkey isn't the same as any of the shortcut keys on Playnite.

I already tried going to Discord's settings then "Game Activity" but the game doesn't appear on the list when I clicked "Add it!".

According to Crow on Discord, they don't really do anything special when launching games (except for Microsoft Store ones), it just starts the executable.

So basically:
Launch a game manually by clicking on it in Explorer: Discord overlay works
Launch the same game through Playnite: No Discord overlay

**Comment by JosefNemec** [comment]
Just a note to anybody that will investigate this: It's possible that it is Discord issue and in that case I don't think it's worth it to implement potential workarounds just to cater to their bugs.

**Comment by JosefNemec** [comment]
I've looked into it and it's super inconsistent in how Discord detects if game is running, nothing we can do about it. #767 should make sure all games are reported properly via Discord's API.
